### ISSUE TITLE
Add documentation for API fields

### ISSUE THREAD

**Issue by Ronoman** [avgupta456/statbotics#302]
As far as I can tell, there isn't an explicit definition for each field in the REST or Python API (or the matching fields in the downloadable CSVs). Many of the fields are self explanatory (`year`, `team`, etc.), but some aren't as simple (`epa_start`, `rp_1_mse`, etc). It would be very useful to have a one or two sentence description of values and derived values, as well as a type and whether `null` is valid for the field.

If this documentation exists and I just haven't found it, feel free to close!

**Comment by Ronoman** [comment]
Well that was quick, it exists here: https://github.com/avgupta456/statbotics/blob/master/docs/API.md. I guess the request is to either link to this or add the corresponding table to one or both API documentation pages on the frontend.

**Comment by avgupta456** [comment]
I agree it should be added to the website. I'll work on getting that done soon.
